docs/node/run-your-node/paratime-client-node.mdx

```diff
@@ -226,19 +193,26 @@ content to the `runtime` part within the `/node/etc/config.yml` file:
 ```yaml
```

Comment:
I'd rename `## Configuring TEE Paratime Client Node` to `### TEE ParaTime Client Node` and move it under Configuration section.